web/sites/default/settings.php

```diff
@@ -122,9 +122,11 @@
    );
 }
 
-### Temp directory
-if (getenv('TMP')) {
-  $config['system.file']['path']['temporary'] = getenv('TMP');
+### Public, private and temporary files paths.
+if (getenv('LAGOON')) {
+  $settings['file_public_path'] = 'sites/default/files';
+  $settings['file_private_path'] = 'sites/default/files/private';  
+  $config['system.file']['path']['temporary'] = 'sites/default/files/private/tmp';
 }
```

Comment:
```suggestion
elseif (getenv('TMP')) {
  $config['system.file']['path']['temporary'] = getenv('TMP');
}
```

Reply:
I experimented with the suggestion feature in GH. I just wasn't sure whether Lagoon might have a purpose for the TMP environment variable.

Reply:
fwiw, I don't think `TMP` is universal... quick check on my host says no.

`sites/default/files/private/tmp` is a good choice because it piggy-backs off the fact that the parent directory is configured as writable. If the platform provider is ok with `sites/default/files/private` then it's good to go.